### PR TITLE
iwv bug fix

### PR DESCRIPTION
**Pull request by HolgerPollyNet**
if water vapor calibration was flagged false, the program still tried to read external data. This is fixed now. 
No water vapor calibration, no attempt to read IWV data !

### CODE REVIEW

File: lib/config/polly_global_config.json [hunk 1]
```diff
@@ -232,24 +241,10 @@
...
-    "Data_Originator_phone": "",
-    "Data_Originator_email": "",
-    "comment": "",
-
     "logbookFile": "",
     "radiosondeFolder": "",
     "calibrationDB": "polly_calibration.db",
     "imgFormat": "png",
     "partnerLabel": "",
-    "prodSaveList": ["overlap", "aerProfFR", "aerProfNR", "aerProfOC", "aerAttBetaFR", "aerAttBetaOC", "aerAttBetaNR", "WVMR_RH", "volDepol", "quasiV1", "quasiV2", "TC", "TCV2", "cloudinfo"]
```
Comment (HolgerPollyNet) [inline lib/config/polly_global_config.json]: should maybe stay?

File: lib/calibration/pollyPolCali.m
Comment (HolgerPollyNet) [inline lib/calibration/pollyPolCali.m:1]: no clue why this is now popping up again

Comment (HolgerPollyNet) [reply lib/calibration/pollyPolCali.m:1]: no clue why all the old changes are popping up, please have a check..

**Review by ulysses78 — COMMENTED** [review]
Can we wait with that till next week?

**Pull request merged**